### PR TITLE
Feature: adds the 'black' code formatter

### PR DESCRIPTION
**Pull request by lsh-0**
* replaces autopep8 with black
* code formatting is only done on python 3.6

all code changes are just automated ones for formatting.

there are improvements I'd like to make to logging and a few bug fixes that would benefit from some clearer formatting beforehand.

### CODE REVIEW

File: .scrub.sh [hunk 1]
```diff
@@ -1,16 +1,10 @@
...
-    --in-place --recursive --aggressive \
-    --ignore E501,E302,E261,E401,E305,E309,E731 \
-    --exclude *.html \
-    ubr/
+if [ -e venv/bin/python3.6 ]; then
+    pip install "black~=19.3b0"
+    black ubr/ --target-version py34
+else
+    # seeing this on new 16.04 lax instances
+    echo "black requires Python 3.6+"
```
Comment (giorgiosironi) [inline .scrub.sh]: this script should never be used on `lax` instances but only during development right? Add `exit 1`?

File: ubr/conf.py [hunk 2]
```diff
@@ -120,23 +141,25 @@ def var(envname, cfgpath, default):
 # so we're explicit about which ones we're using.
 
 AWS = {
-    'aws_access_key_id': cfg('aws.access_key_id'),
-    'aws_secret_access_key': cfg('aws.secret_access_key'),
+    "aws_access_key_id": cfg("aws.access_key_id"),
+    "aws_secret_access_key": cfg("aws.secret_access_key"),
```
Comment (giorgiosironi) [inline ubr/conf.py]: any good reason to stick with one quote `'` or the other `"`, apart from consistency?

File: .scrub.sh (hunk 1 above)
Comment (lsh-0) [reply .scrub.sh]: copy+paste error. I'll fix it up.

File: ubr/conf.py (hunk 2 above)
Comment (lsh-0) [reply ubr/conf.py]: I don't think or have an opinion any more. I defer individual thought regarding formatting in python to Black: https://github.com/psf/black

https://www.thoughtworks.com/radar/techniques/opinionated-and-automated-code-formatting

**Pull request merged**